docs/dev_review.rst

```diff
@@ -1,51 +1,56 @@
-.. libCellML Review Process::
+.. _dev_review:
 
 ==============
-Review Process
+Review process
 ==============
 
-Check for the Green Tick
-========================
+This chapter details the responsibilities of the repository owners, and will help contributors to understand how their code contributions will be assessed and reviewed.
 
-Before accepting a tranche of work into the libCellML prime repository check that `Buildbot <https://buildbot.net/>`_ has tested and passed the code.
+Check for the green tick
+========================
+Before accepting a tranche of work into the libCellML prime repository check that the :libcellml_buildbot:`libCellML Buildbot<>` has tested and passed the code.
 The status of the code is shown in the last commit of a pushed group of commits in the pull request.
 The last commit will have a red cross for a failed build or a green tick for a passed build.
 Obviously, make sure that the last commit has a green tick before merging.
 
-Read the Documentation
+The "Details" link at the bottom of the pull request web page will take you to the build records of the library.
+This page shows the results of the unit tests for each target operating system, the results of the coverage test, the results of the memory check test, and the results of the documentation build.
+
+The coverage statistics for the library (generated from :gcovr:`gcovr<>` ) can be reached through the :ref:`Coverage Statistics <coverage_statistics>` page **TODO** is this still true??
```

Comment:
Yes.

Reply:
Two problems there:
- Don't we always have 100% API coverage as you can't merge without it?  
- And did you look at that page?? This is what it says, which doesn't seem entirely complete somehow!

**TODO**
The output from the coverage testing using gcov is available TODO.

.. note::

   The coverage test sometimes (this behaviour has been observed on `macOS <https://en.wikipedia.org/wiki/MacOS>`_ using `Clang <https://clang.llvm.org/>`_) reports single lines containing only a closing curly brace as not covered.
   This is currently being treated as a false positive.
   This can be seen in the 'Missing' column of the test report where only single lines are reported.
   For the case discussed here, the reported line should only contain a single closing curly brace.
   In this situation, we will accept the missed coverage report.